### PR TITLE
HBG: Favored Enemy and support

### PR DESCRIPTION
**Pull request by Northmoc**
image

### CODE REVIEW

File: forge-game/src/main/java/forge/game/ability/effects/ChooseTypeEffect.java [hunk 1]
```diff
@@ -58,6 +60,14 @@ public void resolve(SpellAbility sa) {
                             }
                         }
                     }
+                } else if (sa.hasParam("MostPrevalentInDefinedDeck")) {
+                    Player definedP = AbilityUtils.getDefinedPlayers(
+                            sa.getHostCard(), sa.getParam("MostPrevalentInDefinedDeck"), sa).get(0);
+                    CardCollectionView deck = definedP.getAllCards();
```
Comment (tool4ever) [inline forge-game/src/main/java/forge/game/ability/effects/ChooseTypeEffect.java:66]: it says library not deck?

Comment (Northmoc) [reply forge-game/src/main/java/forge/game/ability/effects/ChooseTypeEffect.java:66]: I think I've fixed it 😉

Comment (tool4ever) [reply forge-game/src/main/java/forge/game/ability/effects/ChooseTypeEffect.java:66]: params no longer match, you missed a file?

Comment (Northmoc) [reply forge-game/src/main/java/forge/game/ability/effects/ChooseTypeEffect.java]: Yep, just caught it - sorry

**Review by tool4ever — APPROVED** [review]

**Review by paulsnoops — APPROVED** [review]

**Pull request merged**